packages/@webex/plugin-cc/src/AgentLogin/AgentLogin.ts

```diff
@@ -0,0 +1,38 @@
+import {HTTP_METHODS, WebexSDK} from '../types';
+
+export default class AgentLogin {
+  webex: WebexSDK;
+  wccAPIURL: string;
+
+  constructor(webex: WebexSDK, wccAPIURL: string) {
+    this.webex = webex;
+    this.wccAPIURL = wccAPIURL;
+  }
+
+  public async loginAgentWithSelectedTeam(
+    teamId: string,
```

Comment:
Create a type with parameters same as UserStationLogin(In agentx code). Keep the parameter of this methods as data: UserStationLogin and use that data to pass in the API as body